setup/install_upstream_rust_keylime/test.sh

```diff
@@ -23,7 +23,26 @@ rlJournalStart
             rlRun "sed -i 's%/sys/kernel/security/tpm0/binary_bios_measurements%${TPM_BINARY_MEASUREMENTS}%' src/common.rs"
             rlRun "sed -i 's%/sys/kernel/security/tpm0/binary_bios_measurements%${TPM_BINARY_MEASUREMENTS}%' src/main.rs"
         fi
+
+#        rlRun "dnf install gcc -y"
+#        rlRun "dnf install openssl-devel -y"
+#        rlRun "dnf install -y zeromq-devel"
+#        rlRun "export PKG_CONFIG_PATH=/usr/lib64/pkgconfig"
+
+        rlRun "dnf remove -y rust"
+        rlRun "curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --default-toolchain none -y"
+        rlRun "source '$HOME/.cargo/env'"
+        rlRun "rustup default nightly"
+        rlRun "rustup component add llvm-tools-preview"
+        sleep 3
+        #install stuff for code coverage
+        rlRun "cargo install grcov"
+        rlRun "export RUSTFLAGS='-Cinstrument-coverage'"
```

Comment:
to make it work also for systemd services you should define these environment in a unit file